examples/s_app_cp.py

```diff
@@ -63,7 +64,7 @@ def run():
     clients = {'1', '2', '3', '4'}
     apps = [App(loop, n) for n in clients]  # Accept 4 clients with ID's 1-4
     try:
-        loop.run_until_complete(server.run(loop, clients, True))
+        loop.run_until_complete(server.run(loop, clients, True, port=PORT))
```

Comment:
Added the arg for the server too to make this arg more obvious. Also people (like myself) might have a mqtt broker running so only changing the port in local.py is easier than reading the docs to understand where do change the port.